getting-started/vertx/devfile.yaml

```
@@ -0,0 +1,78 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: vertx-http-example
+attributes:
+  persistVolumes: 'false'
+projects:
+  -
+    name: vertx-http-example
+    source:
+      type: git
+      location: "https://github.com/openshift-vertx-examples/vertx-http-example.git"
+components:
+  -
+    type: chePlugin
+    id: redhat/java/latest
+  -
+    type: dockerimage
+    alias: maven
+    image: quay.io/eclipse/che-java8-maven:nightly
+    env:
+      - name: JAVA_OPTS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+          -Duser.home=/home/user"
+      - name: MAVEN_OPTS
+        value: $(JAVA_OPTS)
+    memoryLimit: 512Mi
+    endpoints:
+      - name: '8080/tcp'
+        port: 8080
+    mountSources: true
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+commands:
+  -
+    name: maven build
+    actions:
+      -
+        type: exec
+        component: maven
+        command: "mvn -Duser.home=${HOME} clean install"
```

```suggestion
        command: "mvn clean install"
```
(Assuming https://github.com/eclipse/che-devfile-registry/pull/74 gets approved and merged)